src/test/scala/com/amazon/deequ/analyzers/jdbc/JdbcContextSpec.scala

```diff
@@ -24,7 +24,7 @@ import java.sql.{Connection, DriverManager}
   */
 trait JdbcContextSpec {
 
-  val jdbcUrl = "jdbc:sqlite:memory"
+  val jdbcUrl = "jdbc:sqlite:"
```

Comment:
this is the correct syntax for in-memory databases. created a file called 'memory' before